src/Application/Components/Shop/Shop.scss

```diff
@@ -1,6 +1,6 @@
 .broker-title {
   font-family: 'Playfair Display', serif;
-  font-weight: 900;
   font-size: 40px;
+  font-weight: 900;
```

Comment:
at tamr we combine `font-size` and `font-weight` together in mixins and only ever use the mixins.  That way we *can't* give the app inconsistent font feel, and it helps to categorize font usages together to compare them ("we use `@include textscale-body1` for the list here but `@include textscale-title` for the list there, they really ought to be the same"). Since this app also switches up the font-family, would probably make sense to include those in the mixins as well